### DISCUSSION TITLE
Gantt chart on Kanboard feature

### DISCUSSION THREAD

**Discussion by Pray227** [kanboard/kanboard#5409] [Q&A]
I saw on the kanboard.org page that there is a Gantt chart feature on the dashboard of the Kanboard application like the attached image, but I didn't find this feature in the Kanboard application that I installed. Where can I get the Gantt chart feature? Information please.
Thank You,
![boarddashboard](https://github.com/kanboard/kanboard/assets/60729826/90dd00ef-1b62-41c2-8585-123e155ebc9e)

**Comment by fguillot** [comment]
You need to install an additional plugin to have the Gantt chart, for example https://github.com/kanboard/plugin-gantt or a maintained fork.

**Comment by Pray227** [comment]
I will try to install the additional plugin first. If I get any
difficulties, may I ask you again?
Thank you.
Regards,
Prajudi Setiawan

**Comment by Pray227** [comment]
I have uploaded the plugin-gantt-master.zip file into the Kanban folder on my server as attached, but the Gantt chart feature cannot appear yet. Please enlighten me.
Thank You
![plugin-gantt](https://github.com/kanboard/kanboard/assets/60729826/ef435a7f-c6fa-4663-80f5-4d67dc816918)
![Uploading plugin.PNG…]()

**Comment by Pray227** [comment]
it's already solve. Thx a lot
